APR-Core/QuestHandler.lua

```diff
@@ -925,7 +928,7 @@ local function APR_PrintQStep()
 		if (steps["Bloodlust"] and APR.ZoneTransfer == 0) then
 			if (APR1[APR.Realm][APR.Name]["Settings"]["ShowQList"] == 1) then
 				LineNr = LineNr + 1
-				APR.QuestList.QuestFrames["FS"..LineNr]:SetText("** "..APR_Locals["Bloodlust"].." **")
+				APR.QuestList.QuestFrames["FS"..LineNr]:SetText("** "..APR_LLocals["Bloodlust"].." **")

```

Comment:
Typo : shoulb be `APR.QuestList.QuestFrames["FS"..LineNr]:SetText("** "..L["BLOODLUST "].." **")`